content/css/concepts/colors/terms/rgb/rgb.md

```diff
@@ -36,6 +36,8 @@ h1 {
 }
 ```
 
+![rgb-example-1](https://github.com/troytomasch/docs/blob/css-rgb-images/media/css-rgb-example1.png)
```

Comment:
```suggestion
![RGB White Font Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-rgb-example1.png)
```

Reply:
The files are in the wrong location. Please relocate the files to the filepath noted above